controller/handler_edge_ctrl/common.go

```diff
@@ -377,6 +377,31 @@ func (self *baseSessionRequestContext) verifyIdentityEdgeRouterAccess() {
 	}
 }
 
+func (self *baseSessionRequestContext) verifyEdgeRouterServiceBindAccess() {
+	if self.err == nil {
```

Comment:
I assume this is to skip unnecessary work and that `.err` being set is resolved somewhere else.